stock_split_picking/views/stock_partial_picking.xml

```diff
@@ -8,7 +8,7 @@
         <field name="arch" type="xml">
             <field name ="state" position="before">
                 <button name="split_process"
-                        states="draft,assigned,partially_available"
+                        states="draft,assigned,waiting,confirmed,partially_available"
```

Comment:
no need how in draft, as it trying to do unreserve it

Reply:
but we also want to be able to split a draft picking, even if there is nothing to unreserve, or I misunderstood your comment?

Reply:
ok so do  check before, as when it's in draft and trying to unreserve error appeared